python3/libexec/print-custom-templates

```diff
@@ -20,8 +20,8 @@ def main(argv):
         atexit.register(logout, session)
 
         templates = session.xenapi.VM.get_all_records_where('field "is_a_template" = "true" and field "is_a_snapshot" = "false"' )
-    except:
-        print("Error retrieving template list", file=sys.stderr)
+    except Exception as e:
+        print(f"Error: {e}.Error retrieving template list", file=sys.stderr)
```

Comment:
```suggestion
        print(f"Error retrieving template list: {e}", file=sys.stderr)
```
`{e}` will be the error message of the exception:

The generic introduction of during which operation the Exception happened should be issued first, and then the specific error message from the Exception.

Reply:
It's conventional to use the provided  format not necessarily wrong.

"Error during [operation/action]: [error message]"